_templates/layout.html

```diff
@@ -0,0 +1,41 @@
+{%- extends "alabaster/layout.html" %}
```

Comment:
Yes, I did have sphinx 1.3.1 - but the exclamation point works for this and for sphinx 1.3.2, so I still think it's better.  Maybe a comment about the bug in the layout file to remind us.